aksel.nav.no/website/pages/eksempler/guidepanel/demo.tsx

```diff
@@ -21,4 +21,5 @@ export const Demo = {
 
 export const args = {
   index: 0,
+  desc: "Komponenten blir tilpasset mobil på brekkpunkt sm (<480px). Da blir avataren og paddingen mindre. Avataren flyttes også øverst (poster-variant), hvis ikke du overstyrer det.",
```

Comment:
```suggestion
  desc: "Komponenten blir tilpasset mobil på brekkpunkt sm (<480px). Komponenten blir mer kompakt og Avataren flyttes til toppen (poster-variant), hvis ikke du overstyrer dette.",
```

Reply:
Hmm... I think this will sound like you can override the compact design 🤔 But you can only override the avatar placement (`poster={false}`).

Reply:
What do you think about this?
```suggestion
  desc: "På mobil (brekkpunkt sm) blir komponenten mer kompakt og avataren flyttes til toppen. Avatar-plasseringen kan låses med propen 'poster'.",
```

Reply:
Looks great 🚀